client-v2/src/components/Clipboard.tsx

```diff
@@ -273,7 +269,12 @@ const mergeProps = (
   ...stateProps,
   ...dispatchProps,
   selectArticleFragment: (articleId: string, isSupporting = false) =>
-    dispatchProps.selectArticleFragment(clipboardId, articleId, isSupporting)
+    dispatchProps.selectArticleFragment(
+      articleId,
+      clipboardId,
+      clipboardId,
```

Comment:
Just add a comment to explain passing `clipboardId` twice

Reply:
Done 👍